pkg/controller/util.go

```diff
@@ -863,3 +858,25 @@ func deletePod(req podDeleteRequest) error {
 	}
 	return err
 }
+
+func getPodEnvVar(pvc *v1.PersistentVolumeClaim, ic *ImportController) (*importPodEnvVar, error) {
+	podEnvVar := &importPodEnvVar{}
+	podEnvVar.source = getSource(pvc)
+	podEnvVar.contentType = getContentType(pvc)
+
+	var err error
```

Comment:
ep/secret is only relevant if source != none, if you check here, then you can keep the original logging code in getEndpoint